gear/user_management/src/python/user_app/main.py

```diff
@@ -25,36 +44,34 @@ def create_user(proxy: FlywheelProxy, user_entry: UserDirectoryEntry) -> User:
     Returns:
       the ID for flywheel User created from the directory entry
```

Comment:
This method returns the User object (not ID)

Reply:
fixed